src/modules/75-cd/components/PipelineSteps/Common/Terraform/Editview/TerraformConfigFormHelper.ts

```diff
@@ -0,0 +1,131 @@
+export type ConnectorTypes = 'Git' | 'Github' | 'GitLab' | 'Bitbucket'
+
+export const tfVarIcons: any = {
+  Git: 'service-github',
+  Github: 'github',
+  GitLab: 'service-gotlab',
+  Bitbucket: 'bitbucket'
+}
+
+export const ConnectorMap: Record<string, ConnectorInfoDTO['type']> = {
+  Git: Connectors.GIT,
+  Github: Connectors.GITHUB,
+  GitLab: Connectors.GITLAB,
+  Bitbucket: Connectors.BITBUCKET
+}
+
+export const ConnectorLabelMap: Record<ConnectorTypes, StringKeys> = {
+  Git: 'pipeline.manifestType.gitConnectorLabel',
+  Github: 'common.repo_provider.githubLabel',
+  GitLab: 'common.repo_provider.gitlabLabel',
+  Bitbucket: 'pipeline.manifestType.bitBucketLabel'
+}
+
+export const formInputNames = (isTerraformPlan: boolean) => ({
+  connectorRef: isTerraformPlan
+    ? 'spec.configuration.configFiles.store.spec.connectorRef'
+    : 'spec.configuration.spec.configFiles.store.spec.connectorRef',
+  repoName: isTerraformPlan
+    ? 'spec.configuration.configFiles.store.spec.repoName'
+    : 'spec.configuration.spec.configFiles.store.spec.repoName',
+  gitFetchType: isTerraformPlan
+    ? 'spec.configuration.configFiles.store.spec.gitFetchType'
+    : 'spec.configuration.spec.configFiles.store.spec.gitFetchType',
+  branch: isTerraformPlan
+    ? 'spec.configuration.configFiles.store.spec.branch'
+    : 'spec.configuration.spec.configFiles.store.spec.branch',
+  commitId: isTerraformPlan
+    ? 'spec.configuration.configFiles.store.spec.commitId'
+    : 'spec.configuration.spec.configFiles.store.spec.commitId',
+  folderPath: isTerraformPlan
+    ? 'spec.configuration.configFiles.store.spec.folderPath'
+    : 'spec.configuration.spec.configFiles.store.spec.folderPath'
+})
+/* istanbul ignore next */
+export const formikOnChangeNames = (isTerraformPlan: boolean) => ({
+  repoName: isTerraformPlan
+    ? 'spec.configuration.configFiles.store.spec.repoName'
+    : 'spec.configuration.spec.configFiles.store.spec.repoName',
+  branch: isTerraformPlan
+    ? 'spec.configuration.configFiles.store.spec.branch'
+    : 'spec.configuration.spec.configFiles.store.spec.branch',
+  commitId: isTerraformPlan
+    ? 'spec.configuration.configFiles.store.spec.commitId'
+    : 'spec.configuration.spec.configFiles.spec.store.spec.commitId',
+  folderPath: isTerraformPlan
+    ? 'formik.values.spec.configuration.configFiles.store.spec.folderPath'
+    : 'formik.values.spec.configuration.spec.store.spec.folderPath'
+})
+/* istanbul ignore next */
+export const getBuildPayload = (type: ConnectorInfoDTO['type']) => {
+  if (type === Connectors.GIT) {
+    return buildGitPayload
+  }
+  if (type === Connectors.GITHUB) {
+    return buildGithubPayload
+  }
+  if (type === Connectors.BITBUCKET) {
+    return buildBitbucketPayload
+  }
+  if (type === Connectors.GITLAB) {
+    return buildGitlabPayload
+  }
+  return () => ({})
+}
+
+export const stepTwoValidationSchema = (isTerraformPlan: boolean, getString: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232256)
```suggestion
export const stepTwoValidationSchema = (isTerraformPlan: boolean, getString: unknown) => {
```